tests/validation/tests/v3_api/test_airgap_ha.py

```diff
@@ -0,0 +1,516 @@
+from base64 import b64decode
+import os
+import pytest
+import time
+
+from .common import (
+    ADMIN_PASSWORD,
+    AmazonWebServices,
+    run_command, wait_for_status_code
+)
+from .test_custom_host_reg import RANCHER_SERVER_VERSION
+
+from .test_create_ha import (
+    set_url_and_password,
+    RANCHER_HA_CERT_OPTION,
+    RANCHER_VALID_TLS_CERT,
+    RANCHER_VALID_TLS_KEY
+)
+from lib.aws import AWS_USER
+
+AWS_AMI = os.environ.get("AWS_AMI", "ami-012fd49f6b0c404c7")
+DOCKER_COMPOSE_VERSION = os.environ.get("DOCKER_COMPOSE_VERSION", "1.24.1")
+RKE_VERSION = os.environ.get("RKE_VERSION", "v1.3.0")
+
+# airgap variables
+NUMBER_OF_INSTANCES = int(os.environ.get("RANCHER_AIRGAP_INSTANCE_COUNT", "3"))
+HOST_NAME = os.environ.get('RANCHER_HOST_NAME', "testsa")
+AG_HOST_NAME = HOST_NAME
+RANCHER_AG_INTERNAL_HOSTNAME = AG_HOST_NAME + "-internal.qa.rancher.space"
+RANCHER_AG_HOSTNAME = AG_HOST_NAME + ".qa.rancher.space"
+RANCHER_HELM_EXTRA_SETTINGS = os.environ.get("RANCHER_HELM_EXTRA_SETTINGS")
+
+
+RESOURCE_DIR = os.path.join(os.path.dirname(os.path.realpath(__file__)),
+                            'resource')
+SSH_KEY_DIR = os.path.join(os.path.dirname(os.path.realpath(__file__)),
+                           '.ssh')
+
+# bastion/ registry variables
+PRIVATE_REGISTRY_USERNAME = os.environ.get("RANCHER_BASTION_USERNAME")
+PRIVATE_REGISTRY_PASSWORD = \
+    os.environ.get("RANCHER_BASTION_PASSWORD", ADMIN_PASSWORD)
+RANCHER_EXTERNAL_BASTION = os.environ.get("RANCHER_EXTERNAL_BASTION", "")
+RANCHER_EXTERNAL_HOST_NAME = os.environ.get(
+    "RANCHER_EXTERNAL_HOST_NAME", AG_HOST_NAME)
+REG_HOST_NAME = RANCHER_EXTERNAL_HOST_NAME + "-registry"
+REGISTRY_HOSTNAME = os.environ.get(
+    "REGISTRY_HOSTNAME", REG_HOST_NAME + ".qa.rancher.space")
+
+
+def test_deploy_airgap_rancher(check_hostname_length):
```

Comment:
There's other test that has the same same in `test_airgap.py` we may change this test name to `test_deploy_airgap_rancher_ha`.